packages/expo-linear-gradient/src/NativeLinearGradient.android.tsx

```diff
@@ -1,52 +1,51 @@
 import { requireNativeViewManager } from '@unimodules/core';
-import React from 'react';
+import * as React from 'react';
 import { StyleSheet, View } from 'react-native';
 
-type Props = {
-  colors: number[];
-  locations?: number[] | null;
-  startPoint?: Point | null;
-  endPoint?: Point | null;
-} & React.ComponentProps<typeof View>;
+import { NativeLinearGradientProps } from './NativeLinearGradient.types';
 
-type Point = [number, number];
+const NativeLinearGradient: React.FC<NativeLinearGradientProps> = ({
+  colors,
+  locations,
+  startPoint,
+  endPoint,
+  children,
+  style,
+  ...props
+}: NativeLinearGradientProps) => {
```

Comment:
can merge everything neatly and improve readability over `const x = () =>` with:
```suggestion
export default function NativeLinearGradient({
  colors,
  locations,
  startPoint,
  endPoint,
  children,
  style,
  ...props
}: NativeLinearGradientProps): React.ReactElement {
```